snmp/postfixdetailed

```diff
@@ -500,10 +507,10 @@ my $data=$received."\n".
 	$rardnf."\n".
 	$rarnfqa."\n".
 	$iuscp."\n".
+	$msefl."\n".
```

Comment:
Yes, because the recently committed PR broke it. This syncs it to that.

https://github.com/librenms/librenms/pull/6173